### DISCUSSION TITLE
Disabling Hyperthreading makes things better.

### DISCUSSION THREAD

**Discussion by manatactical** [apohl79/audiogridder#911] [General]
Hey,

I would like to share something that helped me a lot.
My server machine has 18 cores CPU with 196GB, and it's running Vienna Ensemble Pro (core 1 to 14), Cantabile (15 to 17), and Audiogridder (all cores, sandboxing enabled).
Last week, I've disabled Hyperthreading. Since then, all the three software, and especially Audio Gridder, behave way better.

If that is something you have not yet done, consider trying it! 

All the best

**Comment by apohl79** [comment]
Thanks for sharing!

**Comment by CaBuffalo** [comment]
Good to know, thanks.

On Mon, Feb 14, 2022 at 8:06 AM Andreas Pohl ***@***.***>
wrote:

> Thanks for sharing!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/911#discussioncomment-2174333>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6DNOUZBT6CM757UHG3U3ERUJANCNFSM5OLRD6WQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by JamesXXV** [comment]
Thanks for the info :)
